examples/libvirt-caasp.env

```
@@ -0,0 +1,13 @@
+# This is an example configuration file for a SUSE CaaSP deployment
+# on libvirt
+# NOTE: The used image *must* be a SLE-15-SP1 image
+#
+# For a full list of configuration options, see the files in
+# config/*.toml
+
+export ROOKCHECK_HARDWARE_PROVIDER="LIBVIRT"
+export ROOKCHECK_LIBVIRT_CONNECTION="qemu:///system"
+export ROOKCHECK_LIBVIRT_VM_MEMORY="8"
+export ROOKCHECK_DISTRO=SLES_CaaSP
+export ROOKCHECK_NODE_IMAGE_USER=sles
+export ROOKCHECK_LIBVIRT_IMAGE = "http://download.suse.de/install/SLE-15-SP1-JeOS-QU3/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU3.qcow2"
```

the extra spaces around ' = ' need to be removed otherwise this produces a syntax error.

```suggestion
export ROOKCHECK_LIBVIRT_IMAGE="http://download.suse.de/install/SLE-15-SP1-JeOS-QU3/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU3.qcow2"
```